pkg/util/spanlogger/spanlogger.go

```diff
@@ -31,6 +31,17 @@ func New(ctx context.Context, method string, kvps ...interface{}) (*SpanLogger,
 	return logger, ctx
 }
 
+func FromContext(ctx context.Context) *SpanLogger {
+	sp := opentracing.SpanFromContext(ctx)
+	if sp == nil {
+		return noop
```

Comment:
`noop` makes me think that the logger will be noop as well. Can we just remove that variable and just return `&SpanLogger{Logger: util.Logger, Span: noopSpan}`

Reply:
sure !